test/hotspot/jtreg/compiler/vectorization/runner/ArrayTypeConvertTest.java

```diff
@@ -75,6 +108,8 @@ public ArrayTypeConvertTest() {
 
     // ---------------- Integer Extension ----------------
     @Test
+    @IR(failOn = {IRNode.STORE_VECTOR})
```

Comment:
Not sure if it's necessary to fail on the Node, I'm not an expert, so just a question, not a review comment. Similar question for the below  `failOn`s

Reply:
I think it makes sense to add a `failOn` here. We have an RFE that tracks this, and when it is implemented, we should expect this test to "fail", i.e. the vectorization succeeds. Then it would be nice if we can enable a positive IR rule. So I put in a negative now, so we don't forget to put in a positive later.

Reply:
@eme64 I agree with having the negative check but please make the intention clearer in the comment. E.g. "Subword vector casts do not work currently. Assert the vectorization failure so that we are reminded to update the test when this limitation is addressed in the future."

Reply:
That is why I wrote this comment:
`// Subword vector casts do not work currently, see JDK-8342095.`
But I can extend the comment if you like.

Reply:
I see, thanks for explanation!

Reply:
Thanks for extending the comment, Emanuel!